services/API-service/src/api/event/dto/event-place-code.dto.ts

```diff
@@ -50,6 +51,9 @@ export class ActivationLogDto {
   @ApiProperty({ example: 'Maximum alert' })
   public alertClass: string;
 
+  @ApiProperty({ default: AlertLevel.NONE })
+  public alertLevel: AlertLevel;
```

Comment:
is this overlapping with alertClass? Is there opportunity to drop alertClass now?

Reply:
> is this overlapping with alertClass?

I created a mapping from alert level to alert class [here](https://github.com/rodekruis/IBF-system/blob/6c490c442aef00ae742f315ca89f112e9410a22c/services/API-service/src/api/event/event.service.ts#L1137-L1143) to avoid the overlap. `alertClass` seems to be used in many ways so I didn't want to remove it in this PR.

> Is there opportunity to drop alertClass now?

Yes, if we refactor the _many ways_ we use alertClass.

Reply:
Losing it in general may be more difficult, but this use case in particular is about including this in the activation-log, which is just one particular use that we can get rid of right? Given that in practice the values end up the same?

Reply:
Yes, we can get rid of it in the activation log. I'll remove it.

Reply:
done https://github.com/rodekruis/IBF-system/pull/2036/commits/ff1cd23ffcb44249d268fe4d8dccebfdee515934